contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py

```diff
@@ -63,7 +69,7 @@ def _transmit(self, envelopes):
         except Exception as ex:  # TODO: consider RequestException
             logger.warning('Transient client side error %s.', ex)
             # client side error (retryable)
-            return self.options.minimum_retry_interval
+            return ResponseType.RETRY
```

Comment:
Here we might not even received any reponse (e.g. network failure while sending request), it is very confusing to return something called `ResponseType`.